.github/workflows/build.yml

```diff
@@ -32,15 +36,18 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: ensure zstd for cache
```

Comment:
I see you added zstd to the build VM image -- can you please put a comment here prompting to remove this block?

Reply:
I can. When do you think that build will apply, so we can remove it entirely?